lib/resilient/circuit_breaker.rb

```diff
@@ -106,13 +106,20 @@ def reset
     private
 
     def open_circuit
-      @opened_or_last_checked_at_epoch = Time.now.to_i
-      @open = true
+      instrument("resilient.circuit_breaker.open", key: @key) { |payload|
```

Comment:
I think the only issue would be this. It is named the same as https://github.com/jnunemaker/resilient/blob/af76925f65c714a75e927ea536fc00241b4c78d7/lib/resilient/circuit_breaker.rb#L128. Perhaps we call these open_circuit and close_circuit and call it a day? 

I imagine since this was HALF A DECADE ago you've moved on with life. So I'll make the change and merge.